### PR TITLE
Update Invidious to 2022.11.22-5160d8b

### PR DESCRIPTION
**Pull request by ceramicwhite**
Update Invidious to 2022.12.02-5160d8b

### CODE REVIEW

File: invidious/umbrel-app.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ manifestVersion: 1
 id: invidious
 category: Social
 name: Invidious
-version: "2022.09.08-53fb6ad"
+version: "2022.12.02-5160d8b"
```
Comment (nevets963) [inline invidious/umbrel-app.yml:5]: Is this version correct? When I load the app, it's showing: `2022.11.22-5160d8ba`

**Review by nevets963 — COMMENTED** [review]
@ceramicwhite Thanks for the PR to update Invidious!

Tested on `arm64` and `amd64`.

Would you know what is up with the version number mismatch?

**Comment by pynixis** [comment]
Every time I load the application it's `2022.09.08-53fb6ad`

File: invidious/umbrel-app.yml (hunk 1 above)
Comment (ceramicwhite) [reply invidious/umbrel-app.yml]: fixed

**Comment by pynixis** [comment]
Change looks good! Great job everyone.

**Pull request merged**